Dockerfile.arch-rolling

```diff
@@ -0,0 +1,38 @@
+# Dockerfile for Arch rolling release GNAT
+
+FROM archlinux
+
+RUN pacman --noconfirm -Syu bash \
```

Comment:
We need `make` to be installed in the test images. With that, the complete testsuite passes without changes, which is no small feat IMO.